spec/person_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'tests_helper'
+
+require 'json'
+
+describe Person do
+  before :each do
+    @person = Person.new('Age', 'Name', id: 300)
+    @person_obj = { 'id' => 300, 'name' => 'Name', 'age' => 'Age', 'parent_permission' => true }
```

Comment:
- [ ] when testing please be as thorough as possible, for example when writing tests for the `Person` class make sure to check if `can_use_services` equals true or false and make sure to test both positive and negative cases.

Reply:
This issue is well resolved! ✔️